src/components/popover/docs.mdx

```diff
@@ -0,0 +1,82 @@
+---
+componentName: 'Popover'
+---
+
+**🚧 WORK IN PROGRESS 🚧**
+
+This is a rough cut at a Popover component. This implementation is largely duplicative of the tooltip partial in [`@hashicorp/react-logo-grid`](https://github.com/hashicorp/react-components/tree/main/packages/logo-grid/partials/popover-tooltip).
```

Comment:
To add some colour to this, I *think* this `Popover` component may be useful outside of just `dev-popover`. Could certainly use some iteration, though I think it's at least in a decent spot it terms of a11y etc thanks to the underlying `@reach/popover` library.

Reply:
Side question: do you know what the difference is between the Reach UI popover and tooltip? I can't find any docs on their site about popover.

Reply:
Yeah, it's pretty strange they don't have docs on `popover`. IIRC i was working on `logo-grid` when this came up.

From my understanding on the difference - `tooltip` is shown on `hover` (and `focus` i think?), where `popover` is shown on click or tap (or activation via keyboard, not sure what the short name for that is?).